devices/lepton/image14bit/image.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package image14bit implements 14-bit per pixel images.
+//
+// It is compatible with the image/draw package.
+package image14bit
+
+import (
+	"image"
+	"image/color"
+	"image/draw"
+)
+
+// Gray14 represents an image of 14 bit values.
+type Gray14 struct {
+	// Pix holds the image's pixels. Each uint16 element represents one 14-bit
+	// pixel.
+	Pix []uint16
+	// Stride is the Pix stride (in pixels) between vertically adjacent pixels.
+	Stride int
+	// Rect is the image's bounds.
+	Rect image.Rectangle
+}
+
+// NewGray14 returns an initialized Gray14 instance.
+func NewGray14(r image.Rectangle) *Gray14 {
+	w, h := r.Dx(), r.Dy()
+	pix := make([]uint16, w*h)
+	return &Gray14{Pix: pix, Stride: w, Rect: r}
+}
+
+// ColorModel implements image.Image.
+func (i *Gray14) ColorModel() color.Model {
+	return Color14Model
+}
+
+// Bounds implements image.Image.
+func (i *Gray14) Bounds() image.Rectangle {
+	return i.Rect
+}
+
+// Opaque returns whether the image is fully opaque.
+func (i *Gray14) Opaque() bool {
+	return true
+}
+
+// At implements image.Image.
+func (i *Gray14) At(x, y int) color.Color {
+	return i.Color14At(x, y)
+}
+
+// Color14At returns the Color14 value at a point.
```

Comment:
Rename this and SetColor14 to use Intensity instead.

Reply:
Done